app/views/admin/accounts/index.html.erb

```diff
@@ -2,4 +2,28 @@
   <div class="row">
     Accounts
   </div>
+  <table class="table table-bordered table-sm text-center">
+    <thead>
+      <tr>
+        <th>Id</th>
+        <th>Email</th>
+        <th>Role</th>
+        <th>Last sign in</th>
+        <th colspan=3></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @accounts.each do |account| %>
+        <tr class=<%= 'table-danger' if account.unlock_token? %>>
+          <td><%= account.id %></td>
+          <td><%= account.email %></td>
+          <td><span class="<%= account.role.admin? ? 'badge badge-primary' : 'badge badge-info' %>"> <%= account.role %> </span></td>
+          <td><%= account.last_sign_in_at.strftime("%B %d, %Y") if account.last_sign_in_at %></td>
```

Comment:
@gfedorenko No strftime please. Use `Time.current.to_s(:long)`. Change `account.last_sign_in_at` to `account.last_sign_in_at?`.